src/api/factories/ImageFactory.ts

```diff
@@ -57,6 +57,8 @@ export class ImageFactory {
             protocol: originalImageData.protocol,
```

Comment:
all the other factories use getModel method to get/create a Model (or actually CreateRequest) out of the data and getMessage to get the Message to send to the network. even though this isn't exactly the definition of Factory pattern and requires some refactoring in the future, and I should have propably done an interface for this, we should use similar naming everywhere so we don't need to guess what the method does. In this case I dont really need to guess, but still we need to be consistent or in a few months we have a codebase where you need to read every single line of code in a function to understand wtf its about or for.